public/css/feedback-view.css

```diff
@@ -0,0 +1,28 @@
+.identification-section {
```

Comment:
Indentation for this file is different from other files (this file has 2 space indentation while the other css files have 4 space indentation)